conf/nginx/hellogov.conf

```diff
@@ -2,14 +2,34 @@ server {
   listen 80;
   listen [::]:80;
   server_name ###DOMAIN_NAME###;
-  return 301 https://$server_name$request_uri; ##how do we account for multiple subdomains like www.
+  return 301 https://www.$server_name$request_uri;
+}
+
+server {
+  listen 80;
+  listen [::]:80;
+  server_name www.###DOMAIN_NAME###;
+  return 301 https://$server_name$request_uri;
+}
+
+server {
+  listen 443 ssl;
+  listen [::]:443 ssl;
+  server_name ###DOMAIN_NAME###;
+  ssl_certificate /etc/letsencrypt/live/###DOMAIN_NAME###/fullchain.pem;
+  ssl_certificate_key /etc/letsencrypt/live/###DOMAIN_NAME###/privkey.pem; 
+
+  add_header Strict-Transport-Security "max-age=31536000";
+
+  return 301 https://www.$server_name$request_uri; 
 }
 
 server {
   listen 443 ssl;
   listen [::]:443 ssl;
-  ssl_certificate /etc/letsencrypt/live/###DOMAIN_NAME###/fullchain.pem;  ## this actually exists at ..../$DOMAIN_NAME/fullchain.pem
-  ssl_certificate_key /etc/letsencrypt/live/###DOMAIN_NAME###/privkey.pem; ##how do we account for multiple subdomains like www.
+  server_name www.###DOMAIN_NAME###;
+  ssl_certificate /etc/letsencrypt/live/www.###DOMAIN_NAME###/fullchain.pem;
+  ssl_certificate_key /etc/letsencrypt/live/www.###DOMAIN_NAME###/privkey.pem;
 
   add_header Strict-Transport-Security "max-age=31536000";
 
```

Comment:
These updated server definitions route http://hellogov.org, http://www.hellogov.org, and https://hellogov.org redirect to https://www.hellogov.org